### PR TITLE
tf2 fit with pandas df

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

File: pyzoo/zoo/orca/learn/pytorch/estimator.py [hunk 1]
```diff
@@ -325,19 +326,21 @@ def fit(self, data, epochs=1, batch_size=32, feature_cols=None, label_cols=None,
 
         :param data: train data. It can be a XShards, Spark Dataframe, PyTorch DataLoader and
                PyTorch DataLoader creator function.
-               If data is an XShards, each partition is a dictionary of  {'x': feature,
-               'y': label}, where feature(label) is a numpy array or a list of numpy arrays.
+               If data is an XShards, each partition can be Pandas DataFrame or a dictionary of
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/pytorch/estimator.py:329]: each partition can be a Pandas DataFrame

File: pyzoo/zoo/orca/learn/pytorch/estimator.py [hunk 2]
```diff
@@ -325,19 +326,21 @@ def fit(self, data, epochs=1, batch_size=32, feature_cols=None, label_cols=None,
...
-               is a Spark DataFrame. Default: None.
+               is a Spark DataFrame or an XShards of Pandas DataFrame. Default: None.
         :param label_cols: Label column name(s) of data. Only used when data is
-               a Spark DataFrame. Default: None.
+               a Spark DataFrame or an XShards of Pandas DataFrame. Default: None.
         :param validation_data: Validation data. XShards, PyTorch DataLoader and PyTorch DataLoader
                creator function are supported.
-               If data is XShards, each partition is a dictionary of  {'x': feature,
-               'y': label}, where feature(label) is a numpy array or a list of numpy arrays.
+               If data is XShards, each partition can be Pandas DataFrame or a dictionary of
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/pytorch/estimator.py:341]: each partition can be a Pandas DataFrame

File: pyzoo/zoo/orca/learn/pytorch/estimator.py [hunk 3]
```diff
@@ -378,17 +385,19 @@ def predict(self, data, batch_size=4, feature_cols=None):
         Predict input data.
 
         :param data: data to be predicted. It can be an XShards or a Spark Dataframe.
-               If it is an XShards, each partition is a dictionary of
+               If it is an XShards, each partition can be Pandas DataFrame or a dictionary of
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/pytorch/estimator.py:388]: each partition can be a Pandas DataFrame

File: pyzoo/zoo/orca/learn/pytorch/estimator.py [hunk 4]
```diff
@@ -414,13 +423,14 @@ def evaluate(self, data, batch_size=32, feature_cols=None, label_cols=None,
 
         :param data: data: evaluation data. It can be an XShards, Spark Dataframe,
                PyTorch DataLoader and PyTorch DataLoader creator function.
-               If data is an XShards, each partition is a dictionary of  {'x': feature,
-               'y': label}, where feature(label) is a numpy array or a list of numpy arrays.
+               If data is an XShards, each partition can be Pandas DataFrame or a dictionary of
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/pytorch/estimator.py:426]: each partition can be a Pandas DataFrame

File: pyzoo/zoo/orca/learn/tf2/estimator.py [hunk 5]
```diff
@@ -172,8 +173,9 @@ def fit(self, data, epochs=1, batch_size=32, verbose=1,
 
         :param data: train data. It can be XShards, Spark DataFrame or creator function which
                returns Iter or DataLoader.
-               If data is XShards, each partition is a dictionary of  {'x': feature,
-               'y': label}, where feature(label) is a numpy array or a tuple of numpy arrays.
+               If data is XShards, each partition can be Pandas DataFrame or a dictionary of
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/tf2/estimator.py:176]: each partition can be a Pandas DataFrame

File: pyzoo/zoo/orca/learn/tf2/estimator.py [hunk 6]
```diff
@@ -264,8 +272,9 @@ def evaluate(self, data, batch_size=32, num_steps=None, verbose=1,
 
         :param data: evaluate data. It can be XShards, Spark DataFrame or creator function which
                returns Iter or DataLoader.
-               If data is XShards, each partition is a dictionary of  {'x': feature,
-               'y': label}, where feature(label) is a numpy array or a tuple of numpy arrays.
+               If data is XShards, each partition can be Pandas DataFrame or a dictionary of
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/tf2/estimator.py:275]: each partition can be a Pandas DataFrame

File: pyzoo/zoo/orca/learn/tf2/estimator.py [hunk 7]
```diff
@@ -346,16 +359,16 @@ def predict(self, data, batch_size=None, verbose=1,
         Predict the input data
 
         :param data: predict input data.  It can be XShards or Spark DataFrame.
-               If data is XShards, each partition is a dictionary of  {'x': feature}, where feature
-               is a numpy array or a tuple of numpy arrays.
+               If data is XShards, each partition can be Pandas DataFrame or a dictionary of
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/tf2/estimator.py:362]: each partition can be a Pandas DataFrame

**Comment by jason-dai** [comment]
Please add unit test for XShards of Pandas Dataframe

**Comment by Le-Zheng** [comment]
jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/5219/

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**